packages/mikado_reborn/src/components/Modal/Modal.vue

```diff
@@ -64,6 +64,15 @@ export default class Modal extends Vue {
   @Prop({ type: Boolean, default: true })
   readonly closeable!: boolean;
 
+  @Watch('opened', { immediate: true })
+  onOpenedChanged(): void {
+    if (this.opened) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'visible';
+    }
+  }
```

Comment:
Cool ! Can we move this to the overlay component ?
I'm not confident in the fact that we always want to block the body for the modal. We don't have all the use cases yet for this component but for overlay it makes it clear we want that.

And please add `document.body.style.overflow = 'visible';` to the destroy hook too, just in case the component get removed without using the internal opened property.

```suggestion
  @Watch('opened', { immediate: true })
  onOpenedChanged(isOpened: boolean): void {
    if (isOpened) {
      document.body.style.overflow = 'hidden';
      return;
    }
    document.body.style.overflow = 'visible';
  }
```

Reply:
un ternaire passerait aussi dans ce cadre